drive/utils.bal

```diff
@@ -662,3 +665,21 @@ isolated function uploadFileUsingByteArray(http:Client httpClient, byte[] byteAr
     }
 }
 
+# Utility function to retrieve File content as a binary content with its MIME type.
+# 
+# + httpClient - Drive client
+# + path - GET URI path
+# + return - `FileContent` if successful, `error` if not.
+isolated function generateRecordFileContent(http:Client httpClient, string path) returns @tainted FileContent|error {
+    http:Response response = <http:Response> check httpClient->get(<@untainted>path);
+    if (response.statusCode is http:STATUS_OK) {
+        FileContent fileContent  = {
```

Comment:
```suggestion
        return {
            content: check response.getBinaryPayload(),
            mimeType: response.getContentType()
        }; 
```
Does this suggestion reduce the code readability?

Reply:
Yeah. This can improve readability. Let's change it.

Reply:
Changed it /Thanks